test-kimai-version.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+# An awk wizard can probably do this whole script in a single line ;)
+MAJVER=$(echo $KIMAI | awk -F "." '{print $1}')
+MINVER=$(echo $KIMAI | awk -F "." '{print $2}')
+
+if test "${MAJVER}" -eq "1" && test "${MINVER}" -lt 11
```

Comment:
better: replace `.` with empty string in the version and then compare first 3 chars. So `1.10.2` is `110` and both `1.11` and ` 1.11.1` will be `111`, which makes that check more failsafe, as you can check for `${version} -lt 111`

Reply:
After testing your solution I saw that at some point in the future this might fail with 2.x - but I guess until then everybody should be on a composer 2 version and the check can be removed anyways